CHANGES.md

```diff
@@ -4,23 +4,21 @@
-* Add required params for scheduled functions by @stnguyen90 in https://github.com/appwrite/appwrite/pull/6148
-* Update the error message for router_domain_not_configured by @stnguyen90 in https://github.com/appwrite/appwrite/pull/6145
-* Fix _APP_EXECUTOR_HOST for upgrades by @stnguyen90 in https://github.com/appwrite/appwrite/pull/6141
-* Change executor hostname back to appwrite-executor by @stnguyen90 in https://github.com/appwrite/appwrite/pull/6160
-* Fix create execution request filter from previous SDK version by @stnguyen90 in https://github.com/appwrite/appwrite/pull/6146
-* Fix: AI Assistant by @Meldiron in https://github.com/appwrite/appwrite/pull/6153
-* Make URL optional for Create Membership API and Serverside Requests by @PineappleIOnic in https://github.com/appwrite/appwrite/pull/6157
-* Fix: v2 functions by @Meldiron in https://github.com/appwrite/appwrite/pull/6142
-* Fix migrations worker by @abnegate in https://github.com/appwrite/appwrite/pull/6116
-* Fix: Global variables by @Meldiron in https://github.com/appwrite/appwrite/pull/6150
-* Fix webhook secret validation and executor path validation by @vermakhushboo in https://github.com/appwrite/appwrite/pull/6162
-* Fix: Untrusted custom domains + auto-ssl by @Meldiron in https://github.com/appwrite/appwrite/pull/6155
-* Update composer.lock by @PineappleIOnic in https://github.com/appwrite/appwrite/pull/6161
+- Fix create phone session abuse key [#6134][https://github.com/appwrite/appwrite/pull/6134]
+- Fix CLI backwards compatibility [#6125](https://github.com/appwrite/appwrite/pull/6125)
+* Override forEachDocument() to skip the cache collection [#6144](https://github.com/appwrite/appwrite/pull/6144)
+* Fix Not Found error when deploying function from git [#6133](https://github.com/appwrite/appwrite/pull/6133)
+* Add required params for scheduled functions [#6148](https://github.com/appwrite/appwrite/pull/6148)
+* Update the error message for router_domain_not_configured [#6145](https://github.com/appwrite/appwrite/pull/6145)
+* Fix _APP_EXECUTOR_HOST for upgrades [#6141](https://github.com/appwrite/appwrite/pull/6141)
+* Change executor hostname back to appwrite-executor [#6160](https://github.com/appwrite/appwrite/pull/6160)
+* Fix create execution request filter from previous SDK version [#6146](https://github.com/appwrite/appwrite/pull/6146)
+* Make URL optional for Create Membership API and Serverside Requests [#6157](https://github.com/appwrite/appwrite/pull/6157)
+* Support for v2 functions [#6142](https://github.com/appwrite/appwrite/pull/6142)
+* Fix migrations worker [#6116](https://github.com/appwrite/appwrite/pull/6116)
+* Fix: Global variables by [#6150](https://github.com/appwrite/appwrite/pull/6150)
+* Fix webhook secret validation and executor path validation [#6162](https://github.com/appwrite/appwrite/pull/6162)
+* Fix: Untrusted custom domains + auto-ssl [#6155](https://github.com/appwrite/appwrite/pull/6155)
```

Comment:
The fixes should be moved to the Fixes section, right?

Reply:
moved 👍